src/main.cpp

```diff
@@ -258,23 +265,17 @@ static void set_allowed_options(OptionList *allowed_options)
 	allowed_options->insert(std::make_pair("map-dir", ValueSpec(VALUETYPE_STRING,
 			_("Same as --world (deprecated)"))));
 	allowed_options->insert(std::make_pair("world", ValueSpec(VALUETYPE_STRING,
-			_("Set world path (implies local game)"))));
+			_("Set world path (implies local game if used with option --go)"))));
 	allowed_options->insert(std::make_pair("worldname", ValueSpec(VALUETYPE_STRING,
-			_("Set world by name (implies local game)"))));
+			_("Set world by name (implies local game if used with option --go)"))));
```

Comment:
Do you think the part in the parentheses is required? I think it's pretty much self-explaining with the `--go` documentation.

Reply:
For me it was not self-explaining, i would have hat least expected that the world selection list pre-selects the world i passed to --worldname or --world, but both parameters do not have any effect if they are not used with --go